sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubBufferedProducerClient.java

```diff
@@ -60,7 +71,45 @@
  *     &#125;&#41;
  *     .buildClient&#40;&#41;;
  * </pre>
- * <!-- end com.azure.messaging.eventhubs.eventhubbufferedproducerclient.instantiation -->
+ * <!-- end com.azure.messaging.eventhubs.eventhubbufferedproducerclient.construct -->
+ *
+ * <p><strong>Sample: Enqueuing and publishing events</strong></p>
+ *
+ * <p>The following code sample demonstrates enqueuing a set of events in the buffered producer. The producer stores
+ * these events in an internal queue and publishes them when
+ * {@link EventHubBufferedProducerClientBuilder#maxWaitTime(Duration)} has elapsed, the buffer is full, or no more
+ * events can fit into a batch.</p>
+ *
+ * <!-- src_embed com.azure.messaging.eventhubs.eventhubbufferedproducerclient.enqueueEvents-iterable -->
+ * <pre>
+ * TokenCredential credential = new DefaultAzureCredentialBuilder&#40;&#41;.build&#40;&#41;;
+ *
+ * &#47;&#47; &quot;&lt;&lt;fully-qualified-namespace&gt;&gt;&quot; will look similar to &quot;&#123;your-namespace&#125;.servicebus.windows.net&quot;
+ * &#47;&#47; &quot;&lt;&lt;event-hub-name&gt;&gt;&quot; will be the name of the Event Hub instance you created inside the Event Hubs namespace.
+ * EventHubBufferedProducerClient client = new EventHubBufferedProducerClientBuilder&#40;&#41;
+ *     .credential&#40;&quot;fully-qualified-namespace&quot;, &quot;event-hub-name&quot;, credential&#41;
+ *     .onSendBatchSucceeded&#40;succeededContext -&gt; &#123;
+ *         System.out.println&#40;&quot;Successfully published events to: &quot; + succeededContext.getPartitionId&#40;&#41;&#41;;
+ *     &#125;&#41;
+ *     .onSendBatchFailed&#40;failedContext -&gt; &#123;
+ *         System.out.printf&#40;&quot;Failed to published events to %s. Error: %s%n&quot;,
+ *             failedContext.getPartitionId&#40;&#41;, failedContext.getThrowable&#40;&#41;&#41;;
+ *     &#125;&#41;
+ *     .buildClient&#40;&#41;;
+ *
+ * List&lt;EventData&gt; events = Arrays.asList&#40;new EventData&#40;&quot;maple&quot;&#41;, new EventData&#40;&quot;aspen&quot;&#41;,
+ *     new EventData&#40;&quot;oak&quot;&#41;&#41;;
+ *
+ * &#47;&#47; Enqueues the events to be published.
+ * client.enqueueEvents&#40;events&#41;;
+ *
+ * &#47;&#47; Seconds later, enqueue another event.
+ * client.enqueueEvent&#40;new EventData&#40;&quot;bonsai&quot;&#41;&#41;;
+ *
+ * &#47;&#47; Causes any buffered events to be flushed before closing underlying connection.
+ * client.close&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.messaging.eventhubs.eventhubbufferedproducerclient.enqueueEvents-iterable -->
```

Comment:
Add see tags at end to:

Builder
the EH package com.azure.messaging.eventhubs